extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -198,14 +209,22 @@ class CornerstoneViewportService
     }
     const initialImageIndex = csViewport.getCurrentImageIdIndex();
     const camera = csViewport.getCamera();
-    return {
+
+    const presentation:Presentation = {
       presentationIds,
       viewportType:
         !viewportType || viewportType === 'stack' ? 'stack' : 'volume',
       properties,
       initialImageIndex,
       camera,
     };
+
+    if(viewportType === 'orthographic') {
+     const slabThickness:number= csViewport.getSlabThickness()? csViewport.getSlabThickness():MINIMUM_SLAB_THICKNESS;
+     presentation.slabThickness = slabThickness;
```

Comment:
we can directly assign property with turnery operator

Reply:
Addressed in the latest commit.